### DISCUSSION TITLE
adding skin - transmissionic

### DISCUSSION THREAD

**Discussion by triksmelb** [haugene/docker-transmission-openvpn#2271] [Show and tell]
For hours I wasn't able to get **transmissionic** skin working with the variables settings. Wanted to share a solution that finally worked for anyone in the same boat.

Here's how to change your Web UI in haugene/docker-transmission-openvpn

1. Download the latest transmissionic with .zip extension [here](https://github.com/6c65726f79/Transmissionic/releases/) 
2. Extract it to a folder on your Docker host (I have it under /config/skins/transmissionic)
3. Add environment variable TRANSMISSION_WEB_HOME and set it to /webui
4. Add a volume mapping to the container: /your_path/config/skins/transmissionic:/webui
5. Start the container
6. You may need to do a full page refresh or use a private tab to test.

thanks to original poster
https://github.com/linuxserver/docker-transmission/issues/55
and creator of transmissionic for this beautiful skin.
https://github.com/6c65726f79/Transmissionic

**Comment by pkishino** [comment]
Feel free to create a PR to add this skin to the container.
or Edit the wiki and add a section.
cheers
